src/views/WatchListView.vue

```diff
@@ -97,42 +90,6 @@ axios
     nextMovie.value = response.data.nextMovie;
     nextMovieId.value = nextMovie.value.nextMovieId;
   });
-axios
```

Comment:
This call was to fetch the members so we could display who added the movie to the watchlist. I'm not sure if you wanted to have that functionality in the new display, but I'm fine either way

Reply:
Getting rid of it since we weren't using it for now, can bring it back in the future if we need it (YAGNI)